docs/source/operations/indy_networkyaml.md

```diff
@@ -0,0 +1,277 @@
+| branch                               | Branch of the repository where the Helm Charts and value files are stored                                        |
+| release_dir                          | Relative path where flux should sync files                                                                       |
+| chart_source                         | Relative path where the helm charts are stored                                                                   |
+| git_push_url                         | Gitops https URL for git push like "github.com/hyperledger-labs/blockchain-automation-framework.git"             |
+| username                             | Username which has access rights to read/write on repository                                                     |
+| password                             | Password of the user which has access rights to read/write on repository                                         |
+| email                                | Email of the user to be used in git config                                                                       |
+| private_key                          | Path to the private key file which has write-access to the git repo                                              |
+
+
+The services field for each organization under `organizations` section of Indy contains list of `services` which could be trustee/steward/endorser
+
+The snapshot of trustee service with example values is below
+```yaml
+      services:
+        trustees:
+        - trustee:
+          name: provider-trustee
+          genesis: true
+```
+
+The fields under `trustee` service are 
+
+| Field       | Description                                              |
+|-------------|----------------------------------------------------------|
+| name            | Name for the trustee service                 |
+| genesis         | If using domain and pool transaction genesis  |
+
+The snapshot of steward service example values is below
+```yaml
+      services:
+        stewards:
+        - steward:
+          name: provider-steward-1
+          type: VALIDATOR
+          genesis: true
+          publicIp: 3.221.78.194
+          node:
+            port: 9711
+            targetPort: 9711
+            ambassador: 9711        # Port for ambassador service
+          client:
+            port: 9712
+            targetPort: 9712
+            ambassador: 9712        # Port for ambassador service
+```
+The fields under `steward` service are
+
+| Field       | Description                                              |
+|-------------|----------------------------------------------------------|
+| name                | Name of the steward service                     |
+| type      | Certificate Subject for NetworkMap service. Subject format can be referred at [OpenSSL Subject](https://www.openssl.org/docs/man1.0.2/man1/openssl-req.html) |
```

Comment:
This is still the old "networkMap". Change this to VALIDATOR and describe the significance, what other values are possible